documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1080 @@
+
+.Prerequisites
+
+If you are migrating a virtual machine with a direct LUN disk, ensure that the nodes in the {virt} destination cluster that the VM is expected to run on can access the backend storage.
+
+include::snip-migrating-luns.adoc[]
+
+endif::[]
+ifdef::ova[]
+= Migrating an Open Virtual Appliance (OVA) source provider
+
+You can migrate Open Virtual Appliance (OVA) files that were created by VMware vSphere as a source provider using the CLI.
+
+include::snippet_ova_tech_preview.adoc[]
+endif::[]
+ifdef::ostack[]
+= Migrating an {osp} source provider
+
+You can migrate an {osp} source provider using the CLI.
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
+====
+endif::[]
+ifdef::cnv[]
+= Migrating a Red Hat {virt} source provider
+
+You can use a Red Hat {virt} provider as both a source provider and destination provider.
+
+Specifically, the host cluster that is automatically added as a {virt} provider can be used as both a source provider and a destination provider.
+
+You can use the CLI to migrate VMs from the cluster that {project-short} is deployed on to another cluster, or from a remote cluster to the cluster that {project-short} is deployed on.
+endif::[]
+
+.Procedure
+. Create a `Secret` manifest for the source provider credentials:
++
+ifdef::vmware[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: vsphere
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  cacert: | <5>
+    <ca_certificate>
+  url: <api_end_point> <6>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the vCenter user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, specify `false` to verify the certificate. Defaults to `false` if not specified. Skipping certificate verification proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed.
+<5> Enter the CA certificate for the server that serves the SDK, either vCenter or ESXi, depending on the configuration of the provider, unless the CA certificate was replaced by a third-party certificate, in which case, enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
+<6> Specify the API end point URL, for example, `https://<vCenter_host>/sdk`.
```

Comment:
```suggestion
<6> Specify the API endpoint URL, for example, `https://<vCenter_host>/sdk`.
```
I think that in our context it's `endpoint` as a single word (also applies to other providers below)
and this would also need to change to include ESXi endpoint